ops/terraform/env/mgmt/packer-iam.tf

```diff
@@ -0,0 +1,120 @@
+resource "aws_iam_policy" "packer_s3" {
+  #FIXME: This policy should be reconsidered before acceptance of BFD-3698. Suggest removal.
```

Comment:
`bfd-mgmt-packer-s3` was moved here as part of the consolidation effort, however, it probably shouldn't exist...